code/mob/input.dm

```diff
@@ -80,8 +80,7 @@
 			src.glide_size = glide
 			next_move = world.time + (running ? 0.5 : 1.5)
 			return (running ? 0.5 : 1.5)
-		if(ishuman(src)) // ugly hack pls replace src.canmove by direct GET_MOB_PROPERTY() call once available
```

Comment:
What's up with this

Reply:
What do you mean? That's the key line of the optimization, it removes the ishuman check.